docs/reference/operators/add-custom-op.md

```diff
@@ -52,6 +216,94 @@ When using CUDA custom ops, to ensure synchronization between ORT's CUDA kernels
 
 For example, see how the afore-mentioned `MyCustomOp` is being launched and how the Session using this custom op is created.
 
+## Wrapping an external inference runtime in a custom operator
+A custom operator can wrap an entire model that is then inferenced with an external API or runtime. This can facilitate the integration of external inference engines or APIs with ONNX Runtime.
+
+As an example, consider the following ONNX model with a custom operator named "OpenVINO_Wrapper". The "OpenVINO_Wrapper" node encapsulates an entire MNIST model in OpenVINO's native model format (XML and BIN data). The model data is serialized into the node's attributes and later retrieved by the custom operator's kernel to build an in-memory representation of the model and run inference with OpenVINO C++ APIs.
+
+<p align="center"><img width="50%" src="https://www.onnxruntime.ai/images/custom_op_wrapper.png" alt="ONNX model of a custom operator wrapping an OpenVINO MNIST model"/></p>
```

Comment:
I added this image to the image/ folder. What is the proper way to get this image to show up in my personal github page?

Reply:
Use relative urls